src/com/trollworks/gcs/spell/Spell.java

```diff
@@ -261,14 +262,14 @@ protected void prepareForLoad(LoadState state) {
         mDuration = isContainer ? "" : getDefaultDuration();
         mPoints = 1;
         mReference = "";
-        mIsVeryHard = false;
+        mDifficulty = SkillDifficulty.H;
         mWeapons = new ArrayList<>();
     }
 
     @Override
     protected void loadAttributes(XMLReader reader, LoadState state) {
         super.loadAttributes(reader, state);
-        mIsVeryHard = reader.isAttributeSet(ATTRIBUTE_VERY_HARD);
+        mDifficulty = reader.isAttributeSet(ATTRIBUTE_VERY_HARD) ? SkillDifficulty.VH : SkillDifficulty.H;
```

Comment:
If you're going to switch this to use the SkillDifficulty enum, then it has to support all possible values, not just H/VH. In addition, this should no longer be an attribute and should be stored like it is for skills. When loading old versions, the attribute needs to be looked at, but for the new version, only the new tag should be considered.

Reply:
Alright, I wasn't sure how to handle backwards compatibility. I assume this would increase the version of the Spell class, yes?

Reply:
Yes

Reply:
You only need to be able to read old versions. The code should only ever write the latest version.